dagger-functions/src/main/java/io/odpf/dagger/functions/udfs/scalar/LinearTrend.java

```diff
@@ -17,14 +19,16 @@
     /**
      * returns the gradient of the best fit line of the list of non-null demand values given the defined time window.
      *
-     * @param timestampsArray       the timestamps array
+     * @param localDateTimeArray    the timestamps array
```

Comment:
Why did you convert timestampsArray to LocalDateTimeArray ? LocalDateTime would have timezone associated. Who gives it.

Reply:
converted type for hopstarttime to localdatetime